superset-frontend/src/explore/components/controls/DatasourceControl.jsx

```diff
@@ -199,43 +214,17 @@ class DatasourceControl extends React.PureComponent {
                 }`}
               />
             </Label>
-          </OverlayTrigger>
-          <TooltipWrapper
-            label="change-datasource"
-            tooltip={t('More dataset related options')}
-            trigger={['hover']}
+          </Tooltip>
+          <Dropdown
+            overlay={datasourceMenu}
+            trigger={['click']}
+            id="datasource_menu"
```

Comment:
Maybe we can get rid of `id` since you added `data-test`?

Reply:
Actually there are some styles for that, in DatasourceControl.less. @kgabryje would you mind seeing if those styles are still necessary/relevant? If they are, this may be the right opportunity to bring them into the component with Emotion Theme variables wired in. One less LESS file!

Reply:
Done, 1 less file less 🙂